data/virt_autotest/guest_params_xml_files/sles_15_sp3_64_kvm_hvm_uefi_with_power_management_x86_64.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<guest>
+    <guest_os_name>sles</guest_os_name>
+    <guest_version>15-sp3</guest_version>
+    <guest_version_major>15</guest_version_major>
+    <guest_version_minor>3</guest_version_minor>
+    <guest_os_word_length>64</guest_os_word_length>
+    <host_hypervisor_uri></host_hypervisor_uri>
+    <host_virt_type>kvm</host_virt_type>
+    <guest_virt_type>hvm</guest_virt_type>
+    <guest_machine_type></guest_machine_type>
+    <guest_arch>x86_64</guest_arch>
+    <guest_name></guest_name>
+    <guest_domain_name></guest_domain_name>
+    <guest_memory>2048,maxmemory=4096</guest_memory>
+    <guest_vcpus>2,maxvcpus=4</guest_vcpus>
+    <guest_cpumodel></guest_cpumodel>
+    <guest_metadata></guest_metadata>
+    <guest_xpath></guest_xpath>
+    <guest_installation_automation>autoyast</guest_installation_automation>
+    <guest_installation_autoyast_file>sles_15_plus_kvm_hvm_guest_graphical_x86_64.xml</guest_installation_autoyast_file>
+    <guest_installation_method>location</guest_installation_method>
+    <guest_installation_extra_args>sshd=1#sshpassword=nots3cr3t#console=ttyS0,115200n8#textmode=1</guest_installation_extra_args>
+    <guest_installation_wait></guest_installation_wait>
+    <guest_installation_method_others></guest_installation_method_others>
+    <guest_installation_media>http://openqa.suse.de/assets/repo/SLE-15-SP3-Full-x86_64-Build12345-Media1</guest_installation_media>
+    <guest_installation_fine_grained></guest_installation_fine_grained>
+    <guest_boot_settings>loader=/usr/share/qemu/ovmf-x86_64-ms-code.bin</guest_boot_settings>
+    <guest_secure_boot>true</guest_secure_boot>
+    <guest_os_variant></guest_os_variant>
+    <guest_storage_path></guest_storage_path>
+    <guest_storage_type>disk</guest_storage_type>
+    <guest_storage_format>qcow2</guest_storage_format>
+    <guest_storage_label>gpt</guest_storage_label>
+    <guest_storage_size></guest_storage_size>
+    <guest_storage_others></guest_storage_others>
+    <guest_network_type>bridge</guest_network_type>
+    <guest_network_device>br123</guest_network_device>
```

Comment:
Can you help to add&support the profile with bridge br0(or brN) which if not exist can be configured by your base class functions to bind with NIC connected to public network so that public address guest is able to be created?

Reply:
Firstly, I explained about what is br1 or br0 or br3 in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12363#discussion_r620112759, so there is no need to setup multiple public facing networks/multiple public facing bridge devices and this also really should be prohibited.
So, secondly, this can be supported by using already existing get_netaddr parameter. If guest_netaddr is equal to 'host', this means guest will use host public facing network, so the br0. If br0 does not exist, a customized bridge device provided by user can be used, generally speaking:
if (guest_netaddr eq 'host')
then guest uses host public facing network
if the host public facing network is already configured with br0
then use br0 directly
if no public facing bridge network device is configured,
then use guest_network_device provided by user.

Reply:
Thanks for the explanation.